kroma-chain-ops/genesis/genesis.go

```diff
@@ -62,7 +62,9 @@ func NewL2Genesis(config *DeployConfig, block *types.Block) (*core.Genesis, erro
 		ShanghaiTime:                  config.CanyonTime(block.Time()),
 		CancunTime:                    config.EcotoneTime(block.Time()),
 		EcotoneTime:                   config.EcotoneTime(block.Time()),
-		InteropTime:                   config.InteropTime(block.Time()),
+		// TODO(seolaoh): uncomment this when geth updated
+		// KromaMPTTime:                  config.KromaMPTTime(block.Time()),
```

Comment:
I named it `KromaMptTime`

Reply:
Why not using `KromaMPTTime` following the [go naming convention](https://go.dev/wiki/CodeReviewComments#initialisms) for initialisms?